README.md

```diff
@@ -29,18 +29,18 @@ Include javascript/stylesheets:
     # app/assets/javascripts/pageflow/editor.js
     //= require pageflow/chart/editor
 
-    # app/assets/stylesheets/pageflow/application.scss.css;
+    # app/assets/stylesheets/pageflow/application.ccss.scss;
```

Comment:
Now there's a different typo :smile: You can push to the same branch of your fork to add to the pull request.